<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scacchiera</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="changeColors">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Change colors</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="setBomb">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Set bomb</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bombToggled">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Show bomb</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startMoving">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Timed moving</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
